projects/1.16/assets/earth-reworked/earthworks/lang/zh_cn.json

```
@@ -0,0 +1,292 @@
+  "block.earthworks.polished_blue_slate_wall": "磨制蓝色板岩墙",
+  "block.earthworks.polished_green_slate": "磨制绿色板岩",
+  "block.earthworks.polished_green_slate_slab": "磨制绿色板岩台阶",
+  "block.earthworks.polished_green_slate_stairs": "磨制绿色板岩楼梯",
+  "block.earthworks.polished_green_slate_wall": "磨制绿色板岩墙",
+  "block.earthworks.polished_purple_slate": "磨制紫色板岩",
+  "block.earthworks.polished_purple_slate_slab": "磨制紫色板岩台阶",
+  "block.earthworks.polished_purple_slate_stairs": "磨制紫色板岩楼梯",
+  "block.earthworks.polished_purple_slate_wall": "磨制紫色板岩墙",
+  "block.earthworks.purple_slate": "紫色板岩",
+  "block.earthworks.purple_slate_shingles": "紫色板岩瓦",
+  "block.earthworks.purple_slate_shingles_slab": "紫色板岩瓦台阶",
+  "block.earthworks.purple_slate_shingles_stairs": "紫色板岩瓦楼梯",
+  "block.earthworks.purple_slate_shingles_wall": "紫色板岩瓦墙",
+  "block.earthworks.purple_slate_slab": "紫色板岩台阶",
+  "block.earthworks.purple_slate_stairs": "紫色板岩楼梯",
+  "block.earthworks.purple_slate_tiles": "紫色板岩瓦片",
+  "block.earthworks.purple_slate_tiles_slab": "紫色板岩瓦片台阶",
+  "block.earthworks.purple_slate_tiles_stairs": "紫色板岩瓦片楼梯",
+  "block.earthworks.purple_slate_tiles_wall": "紫色板岩瓦片墙",
+  "block.earthworks.purple_slate_wall": "紫色板岩墙",
+  "block.earthworks.rammed_earth": "夯土",
+  "block.earthworks.rammed_earth_slab": "夯土台阶",
+  "block.earthworks.rammed_earth_stairs": "夯土楼梯",
+  "block.earthworks.rammed_earth_wall": "夯土墙",
+  "block.earthworks.reed": "芦苇",
+  "block.earthworks.reed_slab": "芦苇台阶",
+  "block.earthworks.reed_stairs": "芦苇楼梯",
+  "block.earthworks.reed_wall": "芦苇墙",
+  "block.earthworks.sand_gabion": "沙子铁笼",
+  "block.earthworks.sand_gabion_slab": "沙子铁笼台阶",
+  "block.earthworks.sand_gabion_stairs": "沙子铁笼楼梯",
+  "block.earthworks.sand_gabion_wall": "沙子铁笼墙",
+  "block.earthworks.spruce_timber": "云杉木材",
+  "block.earthworks.spruce_timber_slab": "云杉木材台阶",
+  "block.earthworks.spruce_timber_stairs": "云杉木材楼梯",
+  "block.earthworks.spruce_timber_wall": "云杉木材墙",
+  "block.earthworks.spruce_wood_shakes": "云杉木粗瓦",
+  "block.earthworks.spruce_wood_shakes_slab": "云杉木粗瓦台阶",
+  "block.earthworks.spruce_wood_shakes_stairs": "云杉木粗瓦楼梯",
+  "block.earthworks.spruce_wood_shakes_wall": "云杉木粗瓦墙",
+  "block.earthworks.thatch": "茅草",
+  "block.earthworks.thatch_slab": "茅草台阶",
+  "block.earthworks.thatch_stairs": "茅草楼梯",
+  "block.earthworks.thatch_wall": "茅草墙",
+  "block.earthworks.vertical_acacia_fence": "竖纹金合欢栅栏",
+  "block.earthworks.vertical_acacia_planks": "竖纹金合欢木板",
+  "block.earthworks.vertical_acacia_slab": "竖纹金合欢台阶",
+  "block.earthworks.vertical_acacia_stairs": "竖纹金合欢楼梯",
+  "block.earthworks.vertical_birch_fence": "竖纹白桦栅栏",
+  "block.earthworks.vertical_birch_planks": "竖纹白桦木板",
+  "block.earthworks.vertical_birch_slab": "竖纹白桦台阶",
+  "block.earthworks.vertical_birch_stairs": "竖纹白桦楼梯",
+  "block.earthworks.vertical_dark_oak_fence": "竖纹深色橡栅栏",
+  "block.earthworks.vertical_dark_oak_planks": "竖纹深色橡木板",
+  "block.earthworks.vertical_dark_oak_slab": "竖纹深色橡台阶",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "block.earthworks.vertical_dark_oak_slab": "竖直深色橡木台阶",
```
我不太确定这里是不是竖直，但是至少少了个木字。还麻烦进游戏检查一下，还有下面的竖纹楼梯也许要检查。

---

已补充“木”字。
游戏内材质为竖纹，扁平形状台阶。